src/main/resources/mixins.GalacticraftCore.early.json

```diff
@@ -0,0 +1,9 @@
+{
+    "required": true,
+    "minVersion": "0.8.3-GTNH",
+    "package": "micdoodle8.mods.galacticraft.core.mixins",
+    "refmap": "mixins.hodgepodge.refmap.json",
```

Comment:
this should be `GalacticraftCore` @glowredman

Reply:
Fixed in https://github.com/GTNewHorizons/Galacticraft/tree/fix-mixin-exception